dev/release/verify-release-candidate.sh

```diff
@@ -468,6 +469,16 @@ install_conda() {
   conda activate base
 }
 
+install_maven() {
+  show_info "Installing Maven version ${MAVEN_VERSION}..."
```

Comment:
Could you install only when system Maven is older than 3.8.7?
See `install_nodejs()` how to implement it.